share/revng/test/tests/abi/verify-abi-test-binaries.sh

```diff
@@ -28,6 +28,7 @@ revng \
     analyze \
     ImportBinary \
     "$BINARY" \
+    --use-pdb="${BINARY}.pdb" \
```

Comment:
What's the effect of this?
What was the problem before?

Reply:
> What's the effect of this?

This flag explicitly says "if you need a `pdb` when importing a PE binary, instead of the encoded path (build directory of `test/revng-qa`) look wherever the binary is - the `pdb` is named the same but with a suffix".

> What was the problem before?

Since this was only ever working on `elf` binaries, `pdb`s were not relevant.